tasks/deployCollector.ts

```diff
@@ -79,11 +79,17 @@ export const deployCollector = async (
   },
   {});
 
+  const collectorTokens = await collector.getTokens();
+  const tokensToPrint = {};
+  collectorTokens.forEach((tokenAddress, i) => {
+    Object.assign(tokensToPrint, { [`Collector Token ${i}`]: tokenAddress });
+  });
```

Comment:
I'd do this immutably:
```suggestion
const tokensToPrint = (await collector.getTokens()).reduce( (acc, token, i) => {
  acc[`Collector Token ${i}`] = tokenAddress;
  
  return acc;
} , {});
```

Reply:
done